test/helper/launcher/launcher_test.go

```diff
@@ -0,0 +1,35 @@
+package launcher_test
+
+import (
+	"testing"
+	"time"
+
+	_ "embed"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/test/helper/control"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/test/helper/launcher"
+)
+
+//go:embed test.yml
+var testYml string
+
+const (
+	testVersion = "2.1.0"
+)
+
+func TestLaunch(t *testing.T) {
+	control.SkipTestUnless(t, "AKO_LAUNCHER_TEST")
+
+	l := launcher.NewFromEnv(testVersion)
+	assert.NoError(t, l.Launch(
+		testYml,
+		launcher.WaitReady("atlasprojects/my-project", 30*time.Second)))
```

Comment:
as above, this timeout looks pretty aggressive.

Reply:
fix coming